action.yml

```diff
@@ -38,15 +38,23 @@ inputs:
 
   source:
     description: |-
-      (Required, unless providing `metadata` or `image`) Path to source to
-      deploy. If specified, this will deploy the Cloud Run service from the code
-      specified at the given source directory.
+      (Required, unless providing `metadata`, `image`, or `job`) Path to source
+      to deploy. If specified, this will deploy the Cloud Run service from the
+      code specified at the given source directory.
```

Comment:
```suggestion
      (Required, unless providing `metadata`, `image`, or `job`) Path to source
      to deploy. If specified, this will deploy the Cloud Run service or job from the
      code specified at the given source directory.
```